lesson_16/part_2/figure.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+
+
+class Figure(ABC):
+
+    def calculate_perimetr(self):
```

Comment:
I do not see declaration for abstract methods with python decorator. In such case these methods are not interpret  as abstract.